package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cowprotocol/cow-sdk",
-  "version": "5.0.0",
+  "version": "5.0.1",
```

Comment:
Not sure a patch version bump is adequate.
 At least a minor version bump?

Reply:
Sure! Was wondering about this

Reply:
Sure, was wondering about this.